### ISSUE TITLE
schema change for likelihoods for more precision

### ISSUE THREAD

**Issue by istfer** [PecanProject/bety#463]
Related to issue #462 

The SS values I calculate are very close to each other, such as `-35.38833`, `-35.38797` which all become `-35` in the database and can only be retrieved back as `-35`. 

The current table doesn't allow any decimal points for loglikelihood (`numeric(10,0) `):
```
                                       Table "public.likelihoods"
    Column     |              Type              |                        Modifiers                         
---------------+--------------------------------+----------------------------------------------------------
 id            | bigint                         | not null default nextval('likelihoods_id_seq'::regclass)
 run_id        | bigint                         | not null
 variable_id   | bigint                         | not null
 input_id      | bigint                         | not null
 loglikelihood | numeric(10,0)                  | 
 n_eff         | numeric(10,0)                  | 
 weight        | numeric(10,0)                  | 
 residual      | numeric(10,0)                  | 
 created_at    | timestamp(6) without time zone | default utc_now()
 updated_at    | timestamp(6) without time zone | default utc_now()
Indexes:
    "likelihoods_pkey" PRIMARY KEY, btree (id)
    "unique_run_variable_input_combination" UNIQUE CONSTRAINT, btree (run_id, variable_id, input_id)
    "index_likelihoods_on_input_id" btree (input_id)
    "index_likelihoods_on_run_id" btree (run_id)
    "index_likelihoods_on_variable_id" btree (variable_id)
Foreign-key constraints:
    "fk_likelihoods_inputs_1" FOREIGN KEY (input_id) REFERENCES inputs(id)
    "fk_likelihoods_runs_1" FOREIGN KEY (run_id) REFERENCES runs(id)
    "fk_likelihoods_variables_1" FOREIGN KEY (variable_id) REFERENCES variables(id)
Triggers:
    update_likelihoods_timestamp BEFORE UPDATE ON likelihoods FOR EACH ROW EXECUTE PROCEDURE update_timestamp()
```

I need to do more tests for other variables but at least for the `loglikelihood` column I would need 5 digits after the decimal

**Comment by gsrohde** [comment]
@dlebauer, @robkooper Any reason not to use floating point types (e.g., double precisions) for these?  We changed most other numerics to floats in this migration: https://github.com/PecanProject/bety/blob/master/db/migrate/20160303221049_change_numerics_to_floats.rb

**Comment by robkooper** [comment]
I see no reason not to.

**Comment by gsrohde** [comment]
Fixed by release 4.14.
